contracts/strategies/allocation/linear-qf/qf-offchain-spec.md

```diff
@@ -0,0 +1,61 @@
+Spec: Quadratic Funding with Offchain Calculation
+---------------------------------
+
+## Overview 
+Quadratic Funding (QF) is a category of mechanisms that distribute a funding pool based upon individual donations to eligible recipients. Essentially, the amount that an individual donates to a project is considered their "vote", which is weighted according to the type of QF calculation formula being used. 
+
+At time of writing (June 2023), most prominent QF formulas are too computationally expensive to be calculated on-chain at a reasonable cost. As a step towards an eventual fully on-chain QF solution, this strategy functions a hybrid solution: 
+* votes are recorded **on-chain**
+* the distribution is calculated **off-chain**
+* the final distribution is recorded **on-chain**
+
+In order to facilitate this, the contract will emit vote events so that off-chain calculation services can pull in the relevant data. 
+
+## User flows
+Full flow is shown here: 
+![Alt text](<Strategy Evaluation - QF Vote with events.png>)
+
+### Pool Creation
+- identity owner or member creates pool via `createPool` method on `Allo.sol`
+- core QF settings are recorded on allocation strategy contract, including
+    - Application start/end and voting start/end dates
+    - On-chain application requirements 
+    - Sybil Defense settings (minimum contribution, matching cap, donor matching eligibility requirements)
+    - Pool metadata
+
+### Application
+- applicant applies to pool via `applyToPool` on `Allo.sol`
+    - application only processed if application period is open
+    - otherwise, revert
+- allocation strategy checks that application is eligible
+    - if yes, application marked as `Pending`
+    - if no, application marked as `Rejected`
+
+### Application approval
+- pool owner or member makes decision via `reviewApplications` on allocation strategy contract
+- 
```

Comment:
might be worth calling out that
- applications would have to valid entries from the registry 
- every identity can apply a max of 1 time
- the application status can however be toggled

Reply:
> applications would have to valid entries from the registry

I don't think that's true — we decided that the protocol only requires registry identities for creating a pool, right? It's up to the pool owner to decide whether or not they want to require a valid registry entry for application. So that feels like a setting that we could create during pool creation?

Makes sense on the other two points though 🙌